fontc/src/args.rs

```diff
@@ -14,8 +14,17 @@ pub struct Args {
     pub source: PathBuf,
 
     /// Whether to write IR to disk. Must be true if you want incremental compilation.
-    #[arg(short, long, default_value = "true", action = ArgAction::Set)]
-    pub emit_ir: bool,
+    #[arg(
+        short,
+        long,
+        num_args = 0..=1,
+        default_value = "false",
+        default_missing_value = "true",
+        // pre-existing long name kept as alias for backwards compatibility
+        alias = "emit-ir",
+        action = ArgAction::Set,
+    )]
```

Comment:
this seems very complicated for what should be a very common use-case? Like a flag that sets a bool? Is this all about backwards compat? I'm not sure I would bother, or if that's our concern I would maybe just leave the `emit_ir` item and just exit with a helpful message if it gets set?

Reply:
yes, totally about backward compat

@rsheeter said he'd prefer to have the old --emit-ir continue working becuase "I have lots of copy/paste examples with the old name"